scripts/deploy-library.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
I _think_ this can also be removed since it is only referenced by CircleCI config.